### DISCUSSION TITLE
Issue bundling editor into component

### DISCUSSION THREAD

**Discussion by northkode** [outline/rich-markdown-editor#216] [General]
i have added the rich-markdown-editor into my package.json with yarn.

But when i import it and then bundle my main app I am getting an error

```
[!] Error: 'typeOf' is not exported by node_modules/react-is/index.js, imported by node_modules/styled-components/dist/styled-components.esm.js
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
node_modules/styled-components/dist/styled-components.esm.js (1:9)
1: import { typeOf, isElement, isValidElementType } from 'react-is';
```

Even no i can see the react-is does export it, it seems the main library for `rich-markdown-editor` doesnt package in dependencies. 

How can we resolve this for bundling into an application?

**Comment by tommoor** [comment]
I've never used rollup, no idea what would be causing this I'm afraid.

**Comment by northkode** [comment]
I figured out the above issue, however i'm trying to figure out how to bundle this into a single bundle for use on external web apps that can import the whole thing. 

Is this possible? or do i have to build a new bundling script?
